wrappers/obj-c/CMakeLists.txt

```diff
@@ -29,20 +29,24 @@ endif()
 set (PLATFORM_LIBS "")
 # Add flags for obtaining system UUID via IOKit
 if (CMAKE_SYSTEM_NAME STREQUAL "Darwin")
-set (PLATFORM_LIBS "-framework Foundation -framework CoreFoundation -framework IOKit -framework Network")
+set (PLATFORM_LIBS "-framework Foundation -framework CoreFoundation -framework IOKit -framework Network -framework SystemConfiguration")
 endif()
 
 include_directories( 
     ../../lib/include/public/
     ../../lib/include/mat/
+    ../../lib/include/
+    ../../lib/modules/privacyguard/
+    ../../lib/
+    ../../third_party/Reachability/
     )
 
 get_property(dirs DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR} PROPERTY INCLUDE_DIRECTORIES)
 foreach(dir ${dirs})
     message(STATUS "dir='${dir}'")
 endforeach()
 
-set(PLATFORM_FILES main.mm ODWLogManager.mm ODWLogger.mm ODWEventProperties.mm ODWLogConfiguration.mm ODWSemanticContext.mm)
+set(PLATFORM_FILES main.mm ODWLogManager.mm ODWLogger.mm ODWEventProperties.mm ODWLogConfiguration.mm ODWSemanticContext.mm ODWCommonDataContext.mm ODWPrivacyGuardInitConfig.mm ODWPrivacyGuard.mm ../../lib/pal/posix/NetworkInformationImpl.mm ../../third_party/Reachability/ODWReachability.m)
```

Comment:
In one of the previous changes, this file was not updated correctly. As such, the Obj-C sample app was not being built.